src/components/TextInput/TextInput.tsx

```diff
@@ -61,7 +61,7 @@ export type TextInputBaseProps = {
     copyable?: boolean;
     value?: string;
     onChange?: (value: string) => void;
-    onEnterPressed?: (event: KeyboardEvent<HTMLInputElement>) => void;
```

Comment:
Please mark this PR as a breaking change.

Reply:
@imoutaharik yes, I will.  Also I asked from some feedback on current usage.  But of course doesn't change the fact it could cause issues.  Open to discussing this PR's value as well.

Reply:
You'll need to use the `!` otherwise the check for Pr title won't pass.
`fix(TextInput)!: update onEnterPress...`